core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/security/elytron-base/main/module.xml

```diff
@@ -112,5 +111,6 @@
         modules use the parser, they need to have visibility to this module.
         -->
         <module name="org.wildfly.client.config" export="true"/>
+        <module name="org.wildfly.security.elytron-dynamic-ssl" export="true" optional="true"/>
```

Comment:
I am considering if this should not be here.

Maybe the subsystem should have it's own dependency on this new module specified and then in a deployment unit processor associate it with the deployment if running at community level or less.

Reply:
CC @jfdenise , what do you think about this module registration? It should be provisioned only when provisioning with COMMUNITY stability